### PR TITLE
Admin - User CRUD / Design

### PR DESCRIPTION
**Pull request by Eunicegenel**
- Basic styling
- Will try to install bootstrap successfully before the day ends
- See transaction layout
- See user details
- Update User's changeable details through admin ( first name and last name )
- Create new user through admin ( in progress )

image
image
image
image
image

### CODE REVIEW

**Review by lnabrigo18 — APPROVED** [review]

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]
Kindly attach screenshots

File: app/javascript/packs/navbar.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function navbarscroll () {
+  var bodyElement = document.querySelector("body");
+  var navbar = document.getElementById("navbar");
+  this.scrollY > 300 ? navbar.style.opacity = 1 : navbar.style.opacity = 0;
+}
+window.addEventListener("scroll", navbarscroll , false);
```
Comment (analyn-cajocson) [inline app/javascript/packs/navbar.js]: add new line

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/javascript/packs/navbar.js (hunk 1 above)
Comment (Eunicegenel) [reply app/javascript/packs/navbar.js]: line added

**Comment by Eunicegenel** [comment]
> 
> 
> Kindly attach screenshots
SS attatched

File: app/views/users/details.html.erb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          <button type="button" class="btn btn-danger" data-dismiss="modal">Close</button>
+        </div>
+        
+      </div>
+    </div>
+  </div>
+
+  <%= image_tag("logotxt.png", :class => "logo") %>
+  <div class="body-bg"></div>
+</body>
```
Comment (analyn-cajocson) [inline app/views/users/details.html.erb]: add new line

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**